pycbc/vetoes/chisq_cupy.py

```diff
@@ -25,11 +25,12 @@
 import functools
 import numpy
 import cupy as cp
-import lal
 from mako.template import Template
 
+import pycbc.constants
+
 LALARGS = {
-    'TWOPI': lal.TWOPI,
+    'TWOPI': pycbc.constants.TWOPI,
```

Comment:
This *must* be a constant at the point this file is imported? I think that's what happens (see question in constants.py), but if it's not it will be an issue in this kernel.